web_development_101/javascript_basics/project_rock_paper_scissors.md

```diff
@@ -1515,6 +1515,7 @@ Submit a solution with a pull request to this [file](https://github.com/TheOdinP
 - [ebergson's Solution](https://github.com/ebergson/rock-paper-scissors)
 - [Tegristh's Solution](https://github.com/Tegristh/RPS)
 - [jingjing142's Solution](https://github.com/jingjing142/rock-paper-scissors) - [View in Browser](https://jingjing142.github.io/rock-paper-scissors/)
+- [Jonathan Fallas's Solution](https://github.com/Elfjhon/RockPaper) - [View in Browser](https://elfjhon.github.io/RockPaper)
```

Comment:
I corrected your markdown, however, the live link to your solution (corresponding to `View in browser`) goes to a blank page with just a header.

Verify that the page shows what is intended for your solution and comment back.

Reply:
Hi Marvin I already made the changes, the code is running pretty good now.
Thanks for your feedback.